### PR TITLE
Feat/issue 11 fetch json file

### PR DESCRIPTION
**Pull request by hellodebs**

### CODE REVIEW

File: public/api/menu.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "items": [
+    {
+      "id": "1",
+      "image": true,
```
Comment (namirsab) [inline public/api/menu.json:5]: Why is image a boolean? Maybe you should put here the link to the image?

**Review by namirsab — APPROVED** [review]
@hellodebs i love your little PRs. Just one question :)

File: src/pages/Menu.js [hunk 2]
```diff
@@ -1,33 +1,13 @@
...
-    </section>
-  );
+  useEffect(() => {
+    const url = "/api/menu.json";
+    fetch(url)
+      .then((res) => res.json())
+      .then((data) => {
+        console.log(data);
+      });
+  });
```
Comment (mikemangas) [inline src/pages/Menu.js]: shouldnt you catch an error here?

File: public/api/menu.json (hunk 1 above)
Comment (hellodebs) [reply public/api/menu.json:5]: I wanna try to build the url according to the ids myself

**Pull request merged**